### PR TITLE
[Website] Home page and about us page fixes

### PR DESCRIPTION
**Pull request by Codebmk**
#### Summary of Changes (What does this PR do?)
- Removed air quality reading from hero section
- Made hero section mobile responsive
- Made monitor section responsive
- Added new images on about us page
- A continuation of #618 
- Added social links to footer

#### Is this change ready to hit production in its current state?
- [x] Yes
- [ ] No

#### Status of maturity (all need to be checked before merging):

- [x] I consider this code done
- [ ] I've tested this locally
- [ ] This branch has been LGTM'ed by a reviewer

#### How should this be manually tested?
* Do a git fetch then checkout to the `fix-profile-images` branch and run `npm run standalone`

#### Screenshots
hero
monitor

### CODE REVIEW

**Comment by 123MwanjeMike** [comment]
Please note that this PR is a continuation of #618

**Comment by Codebmk** [comment]
> Please note that this PR is a continuation of #618

Yes it is @123MwanjeMike thanks

File: website/frontend/styles/Monitor.scss [hunk 1]
```diff
@@ -104,4 +104,51 @@
       margin: 0;
     }
   }
+}
+
+@media (max-width: $mainBreakPoint) {
```
Comment (Codebmk) [inline website/frontend/styles/Monitor.scss:109]: Adds mobile responsiveness to Monitor section

File: website/frontend/styles/Hero.scss [hunk 2]
```diff
@@ -310,3 +310,82 @@
   }
 
 }
+
+@media(max-width: $mainBreakPoint) {
```
Comment (Codebmk) [inline website/frontend/styles/Hero.scss:314]: Adds mobile responsiveness to Hero section

**Comment by 123MwanjeMike** [comment]
image
image

Looks good, although it takes long to load

**Review by Baalmart — CHANGES_REQUESTED** [review]
Thanks for the changes. We are making great progress here! :)  I have just left a few comments for your review:

1. For the headshot images, thanks for the additions. Did we align with Zana which ones we shall use? 
2. Still on the images, what does this PR mean from the Optimisation standpoint? Aren't we supposed to reference these images from another service? Not sure how the discussion of images is going but gut feeling tells me that something needs to be reviewed here. cc: @123MwanjeMike 
3. Still on the images, wiil there be a corresponding backend for these additions? If yes, is this supposed to be a temporary hacky way of adding these images? cc: @FaithDaka 
4. I am still not getting the expected responsiveness. Perhaps it is my browswer or something. Awaiting your thoughts. Please check all the attached screenshots. 

Thanks again for the works, pretty impressive stuff! :)  @Codebmk 

Screenshot 2022-05-18 at 18 02 20
Screenshot 2022-05-18 at 18 02 35
Screenshot 2022-05-18 at 18 02 56
Screenshot 2022-05-18 at 18 03 09
Screenshot 2022-05-18 at 18 03 28
Screenshot 2022-05-18 at 18 03 45
Screenshot 2022-05-18 at 18 04 06
Screenshot 2022-05-18 at 18 04 22

**Comment by 123MwanjeMike** [comment]
> Thanks for the changes. We are making great progress here! :) I have just left a few comments for your review:
> 
>     1. For the headshot images, thanks for the additions. Did we align with Zana which ones we shall use?
> 
>     2. Still on the images, what does this PR mean from the Optimisation standpoint? Aren't we supposed to reference these images from another service? Not sure how the discussion of images is going but gut feeling tells me that something needs to be reviewed here. cc: @123MwanjeMike
> 
>     3. Still on the images, wiil there be a corresponding backend for these additions? If yes, is this supposed to be a temporary hacky way of adding these images? cc: @FaithDaka
> 
>     4. I am still not getting the expected responsiveness. Perhaps it is my browswer or something. Awaiting your thoughts. Please check all the attached screenshots.
> 
> 
> Thanks again for the works, pretty impressive stuff! :) @Codebmk
> 
> Screenshot 2022-05-18 at 18 02 20
> 
> Screenshot 2022-05-18 at 18 02 35
> 
> Screenshot 2022-05-18 at 18 02 56
> 
> Screenshot 2022-05-18 at 18 03 09
> 
> Screenshot 2022-05-18 at 18 03 28
> 
> Screenshot 2022-05-18 at 18 03 45
> 
> Screenshot 2022-05-18 at 18 04 06
> 
> Screenshot 2022-05-18 at 18 04 22

For the images, I suggest using Cloudinary moving forward. I anticipate many more images will be coming in with time. @Codebmk is that up for consideration?

**Review by Baalmart — COMMENTED** [review]
Hi @Codebmk , please ignore my earlier comments about responsiveness.

At the moment, it is just not responsive on Surface pro., I-pad and other corresponding Tablet variations.

So you can focus on review my other feedback points. Thanks!

**Comment by Baalmart** [comment]
> Hi @Codebmk , please ignore my earlier comments about responsiveness.
> 
> At the moment, it is just not responsive on Surface pro., I-pad and other corresponding Tablet variations.
> 
> So you can focus on review my other feedback points. Thanks!

I had to pull your latest changes. :) @Codebmk

**Comment by Codebmk** [comment]
> > Thanks for the changes. We are making great progress here! :) I have just left a few comments for your review:
> > ```
> > 1. For the headshot images, thanks for the additions. Did we align with Zana which ones we shall use?
> > 
> > 2. Still on the images, what does this PR mean from the Optimisation standpoint? Aren't we supposed to reference these images from another service? Not sure how the discussion of images is going but gut feeling tells me that something needs to be reviewed here. cc: @123MwanjeMike
> > 
> > 3. Still on the images, wiil there be a corresponding backend for these additions? If yes, is this supposed to be a temporary hacky way of adding these images? cc: @FaithDaka
> > 
> > 4. I am still not getting the expected responsiveness. Perhaps it is my browswer or something. Awaiting your thoughts. Please check all the attached screenshots.
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > Thanks again for the works, pretty impressive stuff! :) @Codebmk
> > Screenshot 2022-05-18 at 18 02 20
> > Screenshot 2022-05-18 at 18 02 35
> > Screenshot 2022-05-18 at 18 02 56
> > Screenshot 2022-05-18 at 18 03 09
> > Screenshot 2022-05-18 at 18 03 28
> > Screenshot 2022-05-18 at 18 03 45
> > Screenshot 2022-05-18 at 18 04 06
> > Screenshot 2022-05-18 at 18 04 22
> 
> For the images, I suggest using Cloudinary moving forward. I anticipate many more images will be coming in with time. @Codebmk is that up for consideration?

@123MwanjeMike yes there's a number of options to use. Though sounds like a decision where we need to involve @ja-odur .

**Review by 123MwanjeMike — CHANGES_REQUESTED** [review]
All looks good to me, but is there a way we can speed up the loading by reducing the size of the entrypoint file? 

image

**Comment by Baalmart** [comment]
> > 1. For the headshot images, thanks for the additions. Did we align with Zana which ones we shall use?
> 
> @Baalmart We are using the exterior images

Although I saw people';s pictures in the PR, did not investigate. So thought the PR also uploads the images. :)

**Review by Baalmart — COMMENTED** [review]
Hi @Codebmk , still seeing images in the assets. Is this the intended approach? See attached screenshot for details.
Screenshot 2022-05-20 at 16 49 04

**Review by Baalmart — COMMENTED** [review]
Thanks @Codebmk , all the topics mentioned in the PR description are worked on as mentioned. Thanks!

Just another observation, looks like the famous main.bundle is relatively big.....please review and share thoughts. See attached screenshot. cc: @123MwanjeMike 
Screenshot 2022-05-20 at 17 11 38

**Comment by Baalmart** [comment]
> Thanks for the changes. We are making great progress here! :) I have just left a few comments for your review:
> 
> 1. For the headshot images, thanks for the additions. Did we align with Zana which ones we shall use?
> 2. Still on the images, what does this PR mean from the Optimisation standpoint? Aren't we supposed to reference these images from another service? Not sure how the discussion of images is going but gut feeling tells me that something needs to be reviewed here. cc: @123MwanjeMike
> 3. Still on the images, wiil there be a corresponding backend for these additions? If yes, is this supposed to be a temporary hacky way of adding these images? cc: @FaithDaka
> 4. I am still not getting the expected responsiveness. Perhaps it is my browswer or something. Awaiting your thoughts. Please check all the attached screenshots.
> 
> Thanks again for the works, pretty impressive stuff! :) @Codebmk
> 
> Screenshot 2022-05-18 at 18 02 20
> 
> Screenshot 2022-05-18 at 18 02 35
> 
> Screenshot 2022-05-18 at 18 02 56
> 
> Screenshot 2022-05-18 at 18 03 09
> 
> Screenshot 2022-05-18 at 18 03 28
> 
> Screenshot 2022-05-18 at 18 03 45
> 
> Screenshot 2022-05-18 at 18 04 06
> 
> Screenshot 2022-05-18 at 18 04 22

Thanks @Codebmk ........
Just have one more clarification on the images, will there be a corresponding backend for these image additions for profile and etc? If yes, is this supposed to be a temporary "hacky" way of adding these images? cc: @FaithDaka @ja-odur

**Comment by Codebmk** [comment]
> Thanks @Codebmk ........
Just have one more clarification on the images, will there be a corresponding backend for these image additions for profile and etc? If yes, is this supposed to be a temporary "hacky" way of adding these images? cc: @FaithDaka @ja-odur

Yes @Baalmart it is temporary. There;s more images that are gonna be coming in so for now it seems best to use the current implementation

**Review by 123MwanjeMike — CHANGES_REQUESTED** [review]
The social media links on the footer work, but the twitter links on the individual profiles aren't working

**Review by ja-odur — COMMENTED** [review]
@Codebmk Static files have been removed on the staging branch. Update this PR so that the files changed are clear.

**Review by ja-odur — APPROVED** [review]
LGTM

**Pull request merged**